Automattic/jetpack
# AI Assistant: Hide input when user types on extended block

Fixes #37800

## Proposed changes:

* Adds a listener for the `keydown` event on the block element that hides the input and removes itself

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add an extended block with some content
* Show the input by clicking on "Ask AI Assistant"
* Type something in the input
* Check that it works as expected
* Now focus on the block by clicking anywhere in it
* Check that the input is hidden
* Check that you can show the input again by clicking on "Ask AI Assistant"
